src/components/Modals/Send/views/Details.tsx

```diff
@@ -89,6 +98,7 @@ export const Details = () => {
       </ModalHeader>
       <ModalCloseButton borderRadius='full' />
       <ModalBody>
+        <AccountDropdown assetId={(asset as Asset).assetId} onChange={handleAccountChange} />
```

Comment:
no need to cast here - asset is defined after early return above

Reply:
I believe it does need a cast as is; `asset` is from the form control here and that makes types unhappy

Reply:
resolving per voice convo